analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -443,47 +443,103 @@ def get_compiler_standard(compiler, language):
         return standard
 
     @staticmethod
-    def load_compiler_info(filename, compiler):
+    def load_compiler_info(filename, compiler, c_lang, cpp_lang):
```

Comment:
What is the reason of these two parameters? We could hard-code "c" and "c++" strings in this function.

Reply:
I do not wanted to hardcode it here and in the `ICI.set()` method too. With that there should not be any language key mismatch whet loading information into ICI (from the compiler) or loading the information from the compile info json file.